Lesson07/HW07.StringWorkerTwo/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace HW07.StringWorkerTwo
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            string line = Console.ReadLine();
+            string[] words = line.Split('.', ',',' ', '!',':',';','?');
+
+            int max = 0;
+            foreach (string word in words)
+                if (word.Length > max)
+                    max = word.Length;
+            Console.WriteLine(max);
+
+         
+
+
+
+            Console.ReadKey();
```

Comment:
no comments

Reply:
You must enter a sentence with punctuation.
Necessary :
• Remove the longest word from a string
• Swap the longest word with the shortest word.
• Count how many letters, punctuation marks are in a line.
• Convert the string to a one-dimensional array using a space. Sort the array in descending order - the first index contains the longest word, the last one contains the shortest.

Reply:
Done